script2.js

```diff
@@ -1,15 +1,15 @@
 $(document).ready(function() {
   $('#addTask').click(function() {
-    let $name = $('#usName').val();
+    let $names = $('#usName').val();
```

Comment:
Always use camelCase for the variable naming.

Reply:
changed all variable name to camelCase